shippable (Linux) pycgal build is failing

See: https://app.shippable.com/builds/54e670aa65be240e00671b89

```
[pycgal] Building pycgal/jkohzc6fx4ga, follow log with:
[pycgal] tail -f /home/shippable/.hashdist/tmp/pycgal-jkohzc6fx4ga/build.log
[pycgal|ERROR] Command '[u'/bin/bash', '_hashdist/build.sh']' returned non-zero exit status 1
[pycgal|ERROR] command failed (code=1); raising
make: *** [/home/shippable/workspace/src/github.com/erdc-cm/proteus/linux2/artifact.json] Error 127
```

The shippable profile is here: https://github.com/erdc-cm/proteus/blob/master/.shippable-profile.yaml

--- a/pkgs/pycgal.yaml
+++ b/pkgs/pycgal.yaml
@@ -1,7 +1,7 @@
 extends: [cmake_package]
 
 dependencies:
-  build: [cgal,python]
+  build: [cgal,python,swig]
 
 sources:
 - key: git:c1309859d56a3a4583b6ae2570e5a00dfcdc3035

